src/PrizePool.sol

```diff
@@ -24,10 +24,39 @@ error AlreadyClaimedPrize(address winner, uint8 tier);
 /// @notice Emitted when someone tries to withdraw too many rewards
 error InsufficientRewardsError(uint256 requested, uint256 available);
 
+/// @notice Emitted when an address did not the specified prize on a vault
 error DidNotWin(address winner, address vault, uint8 tier, uint32 prize);
 
+/// @notice Emitted when a proposed fee larger than the max
 error FeeTooLarge(uint256 fee, uint256 maxFee);
 
+/// @notice Emitted when the initialized smoothing number is not less than one
+error SmoothingOutOfBounds(int64 smoothing);
+
+/// @notice Emitted when the contributed amount is more than the available, un-accounted balance
+error ContributionNotFound(uint256 amount, uint256 available);
```

Comment:
`NotFound` is not very descriptive, maybe `ContributionAmountGTDeltaBalance` would be better.
```suggestion
error ContributionAmountGTDeltaBalance(uint256 amount, uint256 available);
```